crates/sound/src/vhu_sound.rs

```diff
@@ -113,14 +233,15 @@ impl VhostUserSoundBackend {
 
         let audio_backend = alloc_audio_backend(config.audio_backend_name)?;
 
+        debug!("everything is ok");
         Ok(Self {
             threads,
             virtio_cfg: VirtioSoundConfig {
                 jacks: 0.into(),
                 streams: 1.into(),
                 chmpas: 0.into(),
             },
-            exit_event: EventFd::new(EFD_NONBLOCK).map_err(Error::EventFdCreate)?,
```

Comment:
Take a look at `lib.rs` on how to define `EventFdCreate`.
As I already suggested, I think we can avoid for now to add the new error enum here, but we can reuse the one in lib.rs

Reply:
Ah yes yes, Ok we can reuse the one in lib.rs